schema/descriptor_test.go

```diff
@@ -232,6 +232,46 @@ func TestDescriptor(t *testing.T) {
 `,
 			fail: true,
 		},
+
+		// expected success: artifactType is present and an IANA compliant value
+		{
+			descriptor: `
+		{
+			"mediaType": "application/vnd.oci.image.manifest.v1+json",
+			"artifactType": "application/vnd.oci.image.manifest.v1+json",
+			"size": 7682,
+			"digest": "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270"
+		}
+		`,
+			fail: false,
+		},
+
+		// expected failure: artifactType does not match pattern (invalid first subtype character)
+		{
+			descriptor: `
+		{
+			"mediaType": "application/vnd.oci.image.manifest.v1+json",
+			"artifactType": "foo/.bar",
+			"size": 7682,
+			"digest": "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270"
+		}
+		`,
+			fail: true,
+		},
+
+		// expected success: data field is present and has base64 content
+		{
+			descriptor: `
+		{
+			"mediaType": "text/plain",
+			"size": 4,
+			"data": "ZmFydHMK",
```

Comment:
I'd prefer we used a different example value here.

Reply:
dang, @sudo-bmitch

Reply:
so official